src/main/resources/schemas/schemaFileSchema.json

```diff
@@ -68,47 +73,84 @@
       },
       "additionalProperties": false
     },
-    "events": {
+    "eventGroups": {
       "type": "array",
       "items": {
+        "title": "Event group",
         "type": "object",
-        "required": [ "name", "schema" ],
-        "additionalProperties": false,
         "properties": {
-          "name": {
+          "eventGroupName": {
```

Comment:
Hmm, it might make it a bit awkward for the two events where the event group is just `DevTools` -- let's table this until we see what Savant comes up with.

Reply:
True, but in that case maybe `eventGroupName` could be empty, or missing.